### PR TITLE
Fix key Metrics flicker from Admin Settings.

### PR DESCRIPTION
**Pull request by sashadoes**
## Summary

Addresses issue:

- #6428 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/settings/SettingsAdmin.js [hunk 1]
```diff
@@ -88,20 +88,11 @@ export default function SettingsAdmin() {
...
-													'google-site-kit'
-												) }
-											</p>
-										</Cell>
-									</Row>
 									<UserInputPreview
 										goTo={ goTo }
 										noHeader
 										noFooter
+										settingsdView
```
Comment (eugene-manuilov) [inline assets/js/components/settings/SettingsAdmin.js:95]:
```suggestion
										settingsView
```

File: assets/js/components/user-input/UserInputPreview.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ export default function UserInputPreview( props ) {
 		submitChanges,
 		error,
 		noHeader,
+		settingsdView = false,
```
Comment (eugene-manuilov) [inline assets/js/components/user-input/UserInputPreview.js:65]:
```suggestion
		settingsView = false,
```

File: assets/js/components/user-input/UserInputPreview.js [hunk 3]
```diff
@@ -154,6 +155,14 @@ export default function UserInputPreview( props ) {
 						{ __( 'Review your answers', 'google-site-kit' ) }
 					</p>
 				) }
+				{ settingsdView && (
```
Comment (eugene-manuilov) [inline assets/js/components/user-input/UserInputPreview.js:158]:
```suggestion
				{ settingsView && (
```

File: assets/js/components/user-input/UserInputPreview.js [hunk 4]
```diff
@@ -242,5 +251,6 @@ UserInputPreview.propTypes = {
 	redirectURL: PropTypes.string,
 	errors: PropTypes.object,
 	noHeader: PropTypes.bool,
+	settingsdView: PropTypes.bool,
```
Comment (eugene-manuilov) [inline assets/js/components/user-input/UserInputPreview.js:254]:
```suggestion
	settingsView: PropTypes.bool,
```

File: assets/js/components/user-input/UserInputPreview.js [hunk 5]
```diff
@@ -154,6 +155,14 @@ export default function UserInputPreview( props ) {
...
 					</p>
 				) }
+				{ settingsdView && (
+					<p className="googlesitekit-settings-user-input__heading">
+						{ __(
+							'Edit your answers for more personalized metrics:',
+							'google-site-kit'
+						) }
+					</p>
+				) }
```
Comment (eugene-manuilov) [inline assets/js/components/user-input/UserInputPreview.js:165]: I think we should preserve the previous layout: this block should go first wrapped with <Row><Cell>, then the div with `googlesitekit-user-input__preview` class.

```js
return (
    <Fragment>
        { settingsView && (
            <Row>
                <Cell>
                    <p ...>...</p>
                </Cell>
             </Row>
        ) }
        <div className={ classnames( 'googlesitekit-user-input__preview', { ... } ) }>
            ...
        </div>
    </Fragment>
)
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @sashadoes. Left a few comments for you. Please, take a look.

**Comment by sashadoes** [comment]
Thanks, @eugene-manuilov . I made an update :)

**Review by eugene-manuilov — APPROVED** [review]
LGTM

**Pull request merged**